sdk/storage/azure-storage-blob/CHANGELOG.md

```diff
@@ -1,7 +1,7 @@
 # Release History
 
-## 12.14.0-beta.2 (Unreleased)
-
+## 12.14.0 (2021-09-15)
+- GA release
```

Comment:
might be worth mentioning that this includes all features from beta. See https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/storage/Azure.Storage.Blobs/CHANGELOG.md#12100-2021-09-08